packages/design-tokens/design-token-previews/components/TokensTable.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import map from "lodash/map";
+import isString from "lodash/isString";
+import {
+  Table,
+  THead,
+  TBody,
+  Tr,
+  Th,
+  Td,
+} from "../../../components/src/components/Table/";
+import { Token } from "../types/Token";
+import { TOKEN_COLUMNS } from "../utils/constants";
+import { buildTokensTableRows } from "../utils";
+
+type TokenTypes = keyof typeof TOKEN_COLUMNS;
+
+export const TokensTable = ({
+  type,
+  data,
+}: {
+  type: TokenTypes;
+  data: Record<string, Record<string, Token>>;
+}): JSX.Element => {
+  const columns = TOKEN_COLUMNS[type];
+  const rows = buildTokensTableRows(columns, data);
+
+  return (
+    <Table style={{ width: "100%" }}>
+      <THead>
+        <Tr>
+          {map(columns, (column) => {
+            return (
+              <Th key={column.name}>
+                <h3>{column.name}</h3>
+              </Th>
+            );
+          })}
+        </Tr>
+      </THead>
+      <TBody>
+        {map(rows, (row) => {
+          return (
+            <Tr key={`${isString(row[0]) ? row[0] : row[0].props.tokenName}`}>
+              {map(row, (cell) => {
+                return (
+                  <Td key={isString(cell) ? cell : `${cell.props.tokenName}`}>
+                    {cell}
+                  </Td>
+                );
+              })}
+            </Tr>
+          );
+        })}
```

Comment:
It generates keys with `undefined`.

![image](https://github.com/Localitos/pluto/assets/1165437/a4411a67-e680-411f-97c9-6b8016998df1)

Relying on `row[0].props.tokenName` seems complicated because it can be any react component and not have the token name.

It may be better to keep only the string on the first column, use it as a key, and put the component here instead. 

```suggestion
        {map(rows, (row) => {
          const tokenName = row[0];
          return (
            <Tr key={tokenName}>
              {map(row, (cell, index) => {
              const column = columns[index];
                return (
                  <Td key={`${tokenName}${column}`}>
                    {index === 0 ? <TokenName name={cell} /> : cell}
                  </Td>
                );
              })}
            </Tr>
          );
        })}
```

I would also change the `Row` type to guarantee that only the last item is a react element.

```typescript
type TokenRow = [...string[], JSX.ELement];
```

Reply:
This one didn't make sense to me how to implement. Because sometimes the key of a color Token could be duplicate and not unique